truffle/contracts/Load/LoadContract.sol

```diff
@@ -268,7 +221,7 @@ contract LoadContract is Ownable {
                     "Token address must be set");
             require(escrow.state == Escrow.State.NOT_CREATED, "Escrow already exists");
         } else {
-            require(_contractedAmount == 0, "Cannot specify a contracted amount for a shipment with no escrow");
+            require(_contractedAmount == 0x0, "Cannot specify a contracted amount for a shipment with no escrow");
```

Comment:
`_contractedAmount` is a uint256 value.  I think this comparison should still be made with `0` instead of the hex value `0x0`.